Projects/Features/SearchFeature/Sources/ViewControllers/WakmusicRecommendViewController.swift

```diff
@@ -0,0 +1,158 @@
+import BaseFeature
+import DesignSystem
+import LogManager
+import PlayListDomainInterface
+import PlaylistFeatureInterface
+import UIKit
+import Utility
+
+final class WakmusicRecommendViewController: BaseReactorViewController<WakmusicRecommendReactor> {
+    private let wmNavigationbarView = WMNavigationBarView().then {
+        $0.setTitle("Hello")
+    }
+
+    private let dismissButton = UIButton().then {
+        let dismissImage = DesignSystemAsset.Navigation.back.image
+        $0.setImage(dismissImage, for: .normal)
+    }
+
+    private lazy var collectionView: UICollectionView = createCollectionView().then {
+        $0.backgroundColor = .black
+    }
+
+    private lazy var dataSource: UICollectionViewDiffableDataSource<RecommendSection, RecommendPlayListEntity> =
+        createDataSource()
+
+    private let playlistDetailFactory: any PlaylistDetailFactory
+
+    init(playlistDetailFactory: any PlaylistDetailFactory, reactor: WakmusicRecommendReactor) {
+        self.playlistDetailFactory = playlistDetailFactory
+        super.init(reactor: reactor)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .white
+        indicator.startAnimating()
+        reactor?.action.onNext(.viewDidLoad)
+    }
+
+    override func addView() {
+        super.addView()
+        self.view.addSubviews(wmNavigationbarView, collectionView)
+    }
+
+    override func setLayout() {
+        super.setLayout()
+
+        wmNavigationbarView.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(STATUS_BAR_HEGHIT())
+            $0.horizontalEdges.equalToSuperview()
+            $0.height.equalTo(48)
+        }
+        wmNavigationbarView.setLeftViews([dismissButton])
+
+        collectionView.snp.makeConstraints {
+            $0.top.equalTo(wmNavigationbarView.snp.bottom)
+            $0.horizontalEdges.equalToSuperview()
+            $0.bottom.equalToSuperview()
+        }
+    }
+
+    override func bind(reactor: WakmusicRecommendReactor) {
+        super.bind(reactor: reactor)
+        collectionView.delegate = self
+    }
+
+    override func bindState(reactor: WakmusicRecommendReactor) {
+        super.bindState(reactor: reactor)
+        let sharedState = reactor.state.share(replay: 2)
```

Comment:
```suggestion
        let sharedState = reactor.state.share()
```
replay 안붙이셔도 돼요. 저희가 생각한 2개까지만 share된다 라는 의미가 아니라 onNext 이후에 구독한 Observer들이 이전 기록을 replay한다는 의미의 replay 더라구요?

Reply:
넵 수정했습니다.

Reply:
수정했습니다.